### ISSUE TITLE
add support for redmine_draw.io

### ISSUE THREAD

**Issue by zuoliang365** [taqueci/redmine_wysiwyg_editor#136]
Hi, 
thanks for your great work. We use visual editor for default editor, and all users feed back that wysiwyg is a greate editor.

Now we installed redmine_draw.io for view and edit diagrams.
And we found that the draw.io tools only exist on wiki/text editor window, not inside the  WYSISYG editor window.

Is there any plan to make WYSIWYG support redmine_drawio ?

Thanks a lot!

**Comment by taqueci** [comment]
Hi @zuoliang365 

For now, I have no plan to support draw.io plugin.
However, I can see the draw.io button on the toolbar!

![image](https://user-images.githubusercontent.com/8342349/140029798-8362e2e8-f7b9-4489-ba98-9e9a46d56062.png)

This has been implemented by:
https://github.com/mikitex70/redmine_drawio/issues/69

I hope you give feedback to draw.io plugin's author.

**Comment by CSCSxx** [comment]
I agree, it would be great if drawio was supported, I can see the toolbar buttons but nothing happens...
